library/pk.c

```diff
@@ -226,6 +226,10 @@ int mbedtls_pk_verify_ext( mbedtls_pk_type_t type, const void *options,
         if( sig_len < mbedtls_pk_get_len( ctx ) )
             return( MBEDTLS_ERR_RSA_VERIFY_FAILED );
 
+#if defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION)
```

Comment:
To make it more unlikely that someone will activate this accidentally, I suggest a check like

    #if FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION == 0x4d4f52454d41471c